engine/src/object.h

```diff
@@ -1491,7 +1491,7 @@ class MCObjectProxy::Handle
     {
         return m_proxy != NULL && m_proxy->ObjectExists();
     }
-    operator bool() const
+    explicit operator bool() const
```

Comment:
This change should probably be removed from this PR - I put it there when:
```
  if (t_object_handle)
   ...
```
Would work - even with it, it appears that the MCObject* cast is used, and not the explicit bool. This is why I used the '.IsValid()' predicate in the ifs rather than relying on casts. @livecodefraser is back now so hopefully can work out why my attempts with 'explicit' failed!